src/assets/styles/base/_typography.scss

```diff
@@ -232,48 +209,58 @@ img {
 
 .intro-paragraph {
   color: $color-grey-dark;
-  font-size: $font-size-sm;
-  line-height: 1.6875em;
+  font-size: $font-size-md;
+  line-height: $line-height;
   margin-bottom: $font-size-lg;
 }
-@media screen and (min-width: map-get($bp, 'md')) {
+@media screen and (min-width: map-get($breakpoint, "breakpoint-3")) {
   .intro-paragraph {
     color: $color-grey-dark;
-    font-size: 1.5rem;
-    line-height: 1.5;
+    font-size: $font-size-h3;
     margin-bottom: $font-size-lg;
   }
 }
 
 blockquote {
-  margin: $spacing-blog-grid auto;
-  padding: 10px 0 10px 40px;
+  margin: $spacing-6 auto;
+  padding: $spacing-2 0 $spacing-2 $spacing-6;
   position: relative;
+  width: 100%;
+
+    @media screen and (min-width: map-get($breakpoint, "breakpoint-2")) {
+      margin: 0;
+      padding: $spacing-3 0 $spacing-4 $spacing-4;
+      width: 75%;
+    }
+
+    @media screen and (min-width: map-get($breakpoint, "breakpoint-4")) {
+      width: 100%;
+    }
 
   q {
     color: $color-grey-dark;
     display: block;
     font-family: $font-family-sans-serif-light;
-    font-size: 1.875em;
-    line-height: 1.375em;
-    margin-bottom: $spacing-vertical-sm;
+    font-size: $font-size-lg;
+    line-height: $line-height - .30;
```

Comment:
Why `- .30`? Also, not that it should have a minus, but if it did, `- .3` would be preferable.